packages/backend/src/open_payments/account/service.ts

```diff
@@ -53,6 +54,7 @@ async function createAccount(
   return await Account.transaction(deps.knex, async (trx) => {
     const account = await Account.query(trx)
       .insertAndFetch({
+        publicName: options.publicName,
```

Comment:
Maybe add a test to check that creation works when a `publicName` is provided.

Reply:
Isn't this test enough to check the creation? https://github.com/omertoast/rafiki/blob/9ee418a32a22debe98ac4528071e069a65d0c0d6/packages/backend/src/open_payments/account/routes.test.ts#L100-L121

Reply:
The test you linked is for the route. The only test for the create method on the service is [this one](https://github.com/omertoast/rafiki/blob/9ee418a32a22debe98ac4528071e069a65d0c0d6/packages/backend/src/open_payments/account/service.test.ts#L62-L69), which doesn't include the public name. I'm suggesting to add a second test that does include a public name to make sure that writing to the DB works.